manifests/repo/debian.pp

```diff
@@ -62,4 +62,35 @@
       }
     }
   }
+
+  if ($php::globals::php_version == '7.1') {
+    # Required packages for PHP 7.1 repository
+    package { 'apt-transport-https':
```

Comment:
Please use [stdlib::ensure_packages](https://github.com/puppetlabs/puppetlabs-stdlib#ensure_packages) for this.